pallets/msa/src/lib.rs

```diff
@@ -133,11 +133,13 @@ pub mod pallet {
 	#[pallet::generate_store(pub(super) trait Store)]
 	pub struct Pallet<T>(_);
 
-	/// Storage type for MSA identifier
+	/// Storage type for the current MSA identifier maximum.
+	/// We need to track this value because the identifier maximum
+	/// is incremented each time a new identifier is created.
 	/// - Value: The current maximum MSA Id
 	#[pallet::storage]
-	#[pallet::getter(fn get_identifier)]
-	pub type MsaIdentifier<T> = StorageValue<_, MessageSourceId, ValueQuery>;
+	#[pallet::getter(fn get_current_msa_identifier_maximum)]
+	pub type CurrentMsaIdentifierMaximum<T> = StorageValue<_, MessageSourceId, ValueQuery>;
```

Comment:
CurrentMsaIdentifierMaximum I do not know what this means.

Reply:
The idea (as discussed with @wilwade) is that since this value is used to create all MSA ids, it can be thought of as the maximum value at any given time (since all MSA ids in the wild are guaranteed to be lower).

Reply:
maybe flight the words...it is not a big deal.

```
get_current_msa_maximum_identifier
```